### PR TITLE
[dashboard] Avoid global min_workers

### PR DESCRIPTION
**Pull request by ijrsvt**
## Why are these changes needed?

* `min_workers` is deprecated and shouldn't be expected to be present.

## Checks

- [ ] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/master/.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

File: dashboard/modules/reporter/reporter_head.py [hunk 1]
```diff
@@ -77,8 +77,8 @@ async def get_ray_config(self, req) -> aiohttp.web.Response:
                     message="Invalid config, could not load YAML.")
 
             payload = {
-                "min_workers": cfg["min_workers"],
-                "max_workers": cfg["max_workers"]
+                "min_workers": cfg.get("min_workers", "unspecified"),
+                "max_workers": cfg.get("max_workers", "unspecified")
```
Comment (AmeerHajAli) [inline dashboard/modules/reporter/reporter_head.py:81]: but we still use the global max_workers though.

Comment (ijrsvt) [reply dashboard/modules/reporter/reporter_head.py:81]: I agree, but I think that stating 'unspecified' is better than this method erroring!

Comment (AmeerHajAli) [reply dashboard/modules/reporter/reporter_head.py:81]: but it should not error though. If it errors then we have a bug. Am I missing something??

**Review by AmeerHajAli — APPROVED** [review]
1 comment.

File: dashboard/modules/reporter/reporter_head.py (hunk 1 above)
Comment (ijrsvt) [reply dashboard/modules/reporter/reporter_head.py:81]: I guess for `max_workers`, yes, but at the end of the day we are **parsing a user-specified file**. I would always rather err on the side of caution here.

**Pull request merged**